### ISSUE TITLE
Testing: #939 Redesigned mnemonic dialog

### ISSUE THREAD

**Issue by fdoving** [RavenProject/Ravencoin#989]
Post your test results as comments to this issue.

Please include release version or commit-hash used when testing.

**Comment by fdoving** [comment]
Consider rewording, as passphrase in this context easily could be mixed up with the  encryption passsphrase.
![image](https://user-images.githubusercontent.com/26197922/121775401-b0abba80-cb87-11eb-8ae9-f41a336ff3e5.png)

**Comment by lsji07** [comment]
Agree. I tested this and ran through several scenarios using windows 10 and Raven Core version v4.7.0.0-b5010492c (64-bit)

I confused the passphrase like @fdoving as above when moving an existing seed through the new seed dialogue screens. I second and recommend rewording that section to highlight the wallet passphrase and seed encryption passphrase are different! 

Otherwise worked perfectly.

**Comment by soullesscomputerboy** [comment]
I just think "passphrase" should be changed to "12 word seedphrase" or similar. Works on linux mint and  core v4.7.0.0-b5010492c (64-bit)

**Comment by m4r1m0** [comment]
Tested and working on Ubuntu 20.04 using raven-4.7.0test3-b5010492c (64 bit). 

Tested all wallet creation/restore options - both with 13th word/passphrase and without. 

Agree that the passphrase box is one of the main sources for confusion amongst QT users and changing the wording to something like "Seed Word"/"Seedphrase" or even "Optional 13th word" would improve user experience.

**Comment by fdoving** [comment]
https://github.com/RavenProject/Ravencoin/pull/1132

**Comment by jeroz1** [comment]
v4.7.0test3-fdov
MacOS Big Sur 11.6.1

**Screen 1:**
String
`Since no wallet.dat file was found in the Raven block chain data directory, a wallet file will be created.`

Shows as:
<img width="564" alt="Screenshot 2021-12-10 at 22 59 21" src="https://user-images.githubusercontent.com/44138303/145646931-66f6fa08-a181-4d34-88fc-97be58e7d5eb.png">

Please fix the space after the comma and write:
`Since no wallet.dat file was found in the Raven block chain data directory, a new wallet file will be created.`

In addition, if I press escape, the window disappears and the wallet is created in the background anyway. Please force people to walk through the guide. That's really confusing and I've already seen people panic because they don't know what their seed words are. 

**Screen 2:**
Passphrase: people REALLY don't know the difference between a passphrase and a password for wallet encryption. They also run into huge difficulties when recovering from seed words while using a passphrase. Please hide the passphrase under an advanced button, and then explain there that it's not a password to encrypt the wallet but an extra seed word.
